modules/OFStateManager/module/src/ofstatemanager.c

```diff
@@ -761,6 +763,7 @@ ind_core_sw_desc_get(of_desc_str_t desc)
     INDIGO_MEM_COPY(desc, ind_core_of_config.desc_stats.sw_desc,
                     OF_DESC_STR_LEN);
 
+    ind_core_bsn_gentable_handler_init();
```

Comment:
shouldn't ind_core_bsn_gentable_handler_init be called from ind_core_init, and not ind_core_sw_desc_get?